agent/util-scripts/gold/test-client-tool-meister/test-53.txt

```diff
@@ -10,25 +10,45 @@
 /var/tmp/pbench-test-utils/pbench/mock-run/0-iter-zero/sample42/tools-default
 /var/tmp/pbench-test-utils/pbench/mock-run/0-iter-zero/sample42/tools-default/testhost.example.com
 /var/tmp/pbench-test-utils/pbench/mock-run/0-iter-zero/sample42/tools-default/testhost.example.com/iostat
+/var/tmp/pbench-test-utils/pbench/mock-run/0-iter-zero/sample42/tools-default/testhost.example.com/iostat/iostat-stderr.txt
```

Comment:
Where did these come from? ISTR they were eliminated in a previous PR (although I can't remember which now). How did they come back? I see nothing in this PR to explain that. Was there a previous deletion or am I imagining things? And was that deletion a mistake?

Reply:
Yes this is a bit confusing, lemme sum up [1]:

PR #1894 mocked out the use of `screen` properly in our unit tests, this removed the direct invocation of `mpstat` and `iostat` in these tests; that PR was implemented independent of this PR because we still don't know if dropping `screen` from the Tool Meister code is a good idea.

[1] https://youtu.be/2sEJmjd8otk

Reply:
Thanks! That is clear :-)